profiles/mishrabiswajit.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/93214676/198717446-cbed943e-382d-4486-a3b4-d02a2d2df8b3.png",
+  "issueId": 1517,
+  "name": "Biswajit Mishra",
+  "username": "mishrabiswajit"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

oh thanks ❤️✌️